app-tasks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
```

Comment:
I don't know whether this is the same as the `slim` version or not, but that one has a smaller footprint and is based on Debian Jessie.